### PR TITLE
Updates to avoid CVE's

### PR DESCRIPTION
**Pull request by dprince**

### CODE REVIEW

**Review by stuggi — APPROVED** [review]
/lgtm

File: go.mod [hunk 1]
```diff
@@ -31,23 +30,34 @@ require (
...
-	kubevirt.io/containerized-data-importer v1.23.5
+	kubevirt.io/containerized-data-importer v1.39.0
 	sigs.k8s.io/controller-runtime v0.9.7
 	sigs.k8s.io/yaml v1.2.0
 )
 
 replace (
+	// required by Microsoft/hcsshim, containers/storage, sriov-network-operator
+	// Not used within this Operator.
+	// Bump to avoid CVE detection with v1.5.4. https://bugzilla.redhat.com/show_bug.cgi?id=1899487, https://bugzilla.redhat.com/show_bug.cgi?id=1982681
```
Comment (bogdando) [inline go.mod:41]: "with 1.5.4" - is prolly a typo meant to be some older version?

Comment (dprince) [reply go.mod]: @bogdando good eye. just pushed a follow up patch to update/fix this comment. Thanks

**Review by stuggi — APPROVED** [review]
/lgtm

**Pull request merged**